openml/study/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .study import OpenMLStudy, OpenMLBenchmarkSuite
+from .study import OpenMLStudy, OpenMLBenchmarkSuite, BaseStudy
```

Comment:
Please do not make `BaseStudy` available to the user here.

Reply:
Removed.
Pushed the changes.

Reply:
Sorry, but this one is still there.

Reply:
Yes, sorry my mistake. Just noticed. Pushed the changes.